cliquenet/src/lib.rs

```diff
@@ -286,6 +309,10 @@ impl Network {
 
     /// Send a message to a party, identified by the given public key.
     pub async fn unicast(&self, to: PublicKey, msg: Bytes) -> Result<()> {
+        if msg.len() > MAX_TOTAL_SIZE {
+            warn!(node = %self.label, %to, len = %msg.len(), "message too large to send");
```

Comment:
Does it matter if this happens? How could we encounter this? Not implying you're doing something wrong, just checking that the log here is not actually potentially a more-severe alert.

Reply:
From the perspective of cliquenet it is just a warning without further consequences, but for the upper layers it is probably severe. For example if nodes can not send their proposals because they are too large, consensus stalls and can not recover. When the bundle format settles we will have to bring back the size calculation to ensure that messages stay below the max. size (whether it is 5 MiB or something else).